packages/ui/src/theme/platforms/react-native/matchers.ts

```diff
@@ -0,0 +1,42 @@
+import { DesignToken } from 'style-dictionary/types/DesignToken';
+
+const isRadius = (token: DesignToken): boolean =>
```

Comment:
nit: Can we alphabetize these functions in the same order that they're exported?